src/ArrayCachePool.php

```diff
@@ -27,9 +27,9 @@ class ArrayCachePool extends AbstractCachePool
     /**
      * @param array $cache
      */
-    public function __construct(array $cache = [])
+    public function __construct(array &$cache = [])
     {
-        $this->cache = $cache;
+        $this->cache = &$cache;
```

Comment:
Arrays are not objects. Arrays are passed by value. I want to be able to do like this: https://github.com/php-cache/array-adapter/blob/master/tests/IntegrationTagTest.php#L23

That is why I need the reference.